frontend/src/components/SamplesList.tsx

```diff
@@ -290,20 +292,18 @@ export default function SamplesList({
             : `${rowCount} matching samples`
         }
         handleDownload={() => setShowDownloadModal(true)}
-        customUI={
+        customUILeft={customToolbarUI}
+        customUIRight={
```

Comment:
Renamed these props to include "left" and "right" to distinguish between components on the toolbar to the left and right of the search bar, respectively.